lib/oli_web/live/delivery/student/learn_live.ex

```diff
@@ -433,6 +447,80 @@ defmodule OliWeb.Delivery.Student.LearnLive do
   attr :unit_raw_avg_score, :map
   attr :assistant_enabled, :boolean, required: true
 
+  # top level page as a card with title and header
+  def unit(%{unit: %{"resource_type_id" => 1}} = assigns) do
```

Comment:
We probably need to think again about all the naming on this LiveView, but it could be a little bit confusing to keep this function component as "`unit`" when it could be also a page. I don't have the ideal name either, but maybe something more generic like `top_level_resource` or a descriptive name that explains more about its structure on the page (such as **row** or **slider**).

Reply:
Yes, I agree. I think `row` is very descriptive considering how the learn page structure is shown.